book/src/keymap.md

```diff
@@ -275,33 +276,51 @@ Mappings in the style of [vim-unimpaired](https://github.com/tpope/vim-unimpaire
-| `Ctrl-k`                | delete to end of line       | `kill_to_line_end`      |
-| `backspace`, `Ctrl-h`   | delete previous char        | `delete_char_backward`  |
-| `delete`, `Ctrl-d`      | delete previous char        | `delete_char_forward`   |
-| `Ctrl-p`, `Up`          | move to previous line       | `move_line_up`          |
-| `Ctrl-n`, `Down`        | move to next line           | `move_line_down`        |
+We support many readline/emacs style bindings in insert mode for
+convenience. These can be helpful for making simple modifications
+without escaping to normal mode, but beware that you will not have an
+undo-able "save point" until you return to normal mode.
+
+| Key                       | Description                 | Command                 |
+| -----                     | -----------                 | -------                 |
+| `Escape`                  | Switch to normal mode       | `normal_mode`           |
+| `Ctrl-x`                  | Autocomplete                | `completion`            |
+| `Ctrl-r`                  | Insert a register content   | `insert_register`       |
+| `Ctrl-w`, `Alt-backspace` | Delete previous word        | `delete_word_backward`  |
+| `Alt-d`                   | Delete next word            | `delete_word_forward`   |
+| `Alt-b`, `Alt-Left`       | Backward a word             | `move_prev_word_end`    |
+| `Ctrl-b`, `Left`          | Backward a char             | `move_char_left`        |
+| `Alt-f`, `Alt-Right`      | Forward a word              | `move_next_word_start`  |
+| `Ctrl-f`, `Right`         | Forward a char              | `move_char_right`       |
+| `Ctrl-e`, `End`           | Move to line end            | `goto_line_end_newline` |
+| `Ctrl-a`, `Home`          | Move to line start          | `goto_line_start`       |
+| `Ctrl-u`                  | Delete to start of line     | `kill_to_line_start`    |
+| `Ctrl-k`                  | Delete to end of line       | `kill_to_line_end`      |
+| `Ctrl-j`, `Enter`         | Insert new line             | `insert_newline`        |
+| `backspace`, `Ctrl-h`     | Delete previous char        | `delete_char_backward`  |
+| `delete`, `Ctrl-d`        | Delete previous char        | `delete_char_forward`   |
+| `Ctrl-p`, `Up`            | Move to previous line       | `move_line_up`          |
+| `Ctrl-n`, `Down`          | Move to next line           | `move_line_down`        |
+| `PageUp`                  | Move one page up            | `page_up`               |
+| `PageDown`                | Move one page down          | `page_down`             |
+| `Alt->`                   | Go to end of buffer         | `goto_file_end`         |
+| `Alt-<`                   | Go to start of buffer       | `goto_file_start`       |
 
 ## Select / extend mode
 
 I'm still pondering whether to keep this mode or not. It changes movement
 commands (including goto) to extend the existing selection instead of replacing it.
```

Comment:
I wonder if it makes sense to rewrite this paragraph as well? We might just be able to remove it now that we have discussion of it in the keymap issue and #1570

Reply:
Can be removed 👍

Reply:
Already did so, just forgot to set this to resolved